vhdbuilder/packer/test/windows-files-check.ps1

```diff
@@ -86,22 +84,26 @@ function Start-Job-To-Expected-State {
     }
 }
 
-function DownloadFileWithRetry {
+function DownloadFile {
     param (
         $URL,
         $Dest,
         $retryCount = 5,
         $retryDelay = 0,
         [Switch]$redactUrl = $false
     )
-    Write-Output "Downloading $URL"
-    curl.exe --silent -f --retry $retryCount --retry-delay $retryDelay -L $URL -o $Dest
-    if ($LASTEXITCODE) {
+
+    Write-Host "Downloading $URL"
+    try {
+        # We don't retry as that could hide flakey upstream servers that will cause issues in prod.
```

Comment:
Why in prod, it doesn't retry the download? What is the concern there?

Reply:
This is a preference thing. I've had some downloads from the upstream systems fail in prod (even with retrys) and would prefer to see them as a failure in agent baker rather than a flaky upstream being hidden by automated retrys. That way we're likely to notice the signals in AgentBaker PRs and can get them fixed before we have a prod incident.

(yes, I have a production incident for something like this :)

The production code, however, should retry.